pages/Configuring/Window-Rules.md

```diff
@@ -1,54 +1,64 @@
-## Table of contents
+# Table of contents
+
 {{< toc >}}
 
-## Window Rules V1
+# Window Rules V1
+
+You can set window rules to achieve different behaviours from the active container.
 
-You can set window rules for various actions. These are applied on window open!
+## Syntax
 
 ```ini
 windowrule=RULE,WINDOW
 ```
 
-`RULE` is a rule (and a param if applicable)
-
-`WINDOW` is a RegEx, either:
-
-- plain regex (for matching a window class)
-- `title:` followed by a regex (for matching a window's title)
+- `RULE` is a rule (and a param if applicable)
+- `WINDOW` is a RegEx, either:
+  - plain RegEx (for matching a window class);
+  - `title:` followed by a regex (for matching a window's title)
 
-you can get both by inspecting `hyprctl clients`
-
-Examples:
+### Examples
 
 ```ini
 windowrule=float,^(kitty)$
 windowrule=move 0 0,title:^(Firefox)(.*)$
 ```
 
-## Window Rules V2
+# Window Rules V2
 
-In order to allow more flexible rules, while also not breaking compatibility with the above
-rule system, window rules v2 were implemented.
+In order to allow more flexible rules, while retaining compatibility with the above
+rule system, window rules V2 were implemented.
 
 In V2, you are allowed to match multiple variables.
 
-the `RULE` field is unchanged, but in the `WINDOW` field, you can put regexes for multiple values like so:
+the `RULE` field is unchanged, but in the `WINDOW` field, you can put regexes
+for multiple values like so:
+
 ```ini
 windowrulev2 = float,class:^(kitty)$,title:^(kitty)$
 ```
 
 For now, the supported fields are:
+
 ```ini
 class - class regex
 title - title regex
 xwayland - 0/1
 floating - 0/1
 ```
 
-Keep in mind you do *not* need to define all of them, but you need to define at least one.
+Keep in mind that you _have_ to declare at least one field, but not all.
+
+{{< hint type=tip >}}
+
+To get more information about a window's class, title, XWayland status or its size;
+you can use `hyprctl clients`.
+
+{{< /hint >}}
 
 ## Rules
 
+=======
```

Comment:
Why

Reply:
@0x4nt forgot to remove the ======= from the merge conflict lolmao